### PR TITLE
Three more new toys for the arcades!

### PR DESCRIPTION
**Pull request by itseasytosee**
## About The Pull Request

Adds three more toys to the pool for arcade cabinets.

**Trick Blindfold** A clever trick that allows you to cheat at games like pin the security baton on the clown, or pinata games

**Squeaky Brain** A toy that looks like a brain that makes a squishing sound. (uses old brain sprite)

Last but not least (personal favorite)

**Broken Radio**
An old radio that can't pick up anything but static.
It plays "static" which is radiation ticking for about twelve seconds when used (with a reasonable cooldown)
Used a remodeled Geiger counter sprite.

## Why It's Good For The Game
Some new pranking items sounded fun to me. The radio is bound to give some engi a heart attack.

## Changelog
:cl: itseasytosee
add: Three more toys to find in your local arcade cabinet! Squeaky brain, trick blindfold, and a broken radio. Collect em all!
/:cl:

### CODE REVIEW

**Comment by itseasytosee** [comment]
holy shit commits

**Comment by itseasytosee** [comment]
file ./code/modules/clothing/glasses/_glasses.dm is missing a trailing newline

**Comment by trollbreeder** [comment]
Broken radio is awesome, i'd love it if this gets merged

**Comment by Shadowflame909** [comment]
Prisoners trick sec into thinking the ai is flooding tritium, and to let them out.

**Comment by Bawhoppen** [comment]
I think the geiger counter noise should come from something else... since radio static is kinda too loose of a connection in my opinion
Though, I'm at a loss for what it could be replaced with, other than maybe some broken kid's toy instrument or regenerative bluespace bubblewrap

**Comment by itseasytosee** [comment]
Yeah, also I want the item to at least to resemble the geiger counter and radio just seemed the most logical. I can't think of anything else other than fake geiger counter which I feel is cheating

File: code/modules/research/designs/misc_designs.dm [hunk 1]
```diff
@@ -1,543 +1,543 @@
-
```
Comment (81Denton) [inline code/modules/research/designs/misc_designs.dm]: Unless I'm missing something, you don't need to change anything about this file and probably did it by accident. Revert the changes and this is good to be merged.

**Review by 81Denton — CHANGES_REQUESTED** [review]

**Comment by itseasytosee** [comment]
Alright, so my original repo does not exist anymore so I think I'm just going to redo this one
